MHTimer.js

```diff
@@ -2450,6 +2484,39 @@ function integerComma(number) {
     return number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
 }
 
+/**
+ * Reset the Relic Hunter location so reminders know to update people
+ */
+function resetRH() {
+    console.log(`Relic hunter location was ${relic_hunter.location} from ${relic_hunter.source}`);
+    relic_hunter.location = 'unknown';
+    relic_hunter.source = 'reset';
+    rescheduleResetRH();
+    console.log('Relic hunter location was reset');
+}
+
+/**
+ * Continue resetting Relic Hunter location
+ */
+
+function rescheduleResetRH() {
+    if (relic_hunter.timeout)
+        clearTimeout(relic_hunter.timeout);
+
+    relic_hunter.timeout = setTimeout(resetRH, Interval.fromDateTimes(DateTime.utc().plus({milliseconds: 10}),DateTime.utc().endOf('day')).length('milliseconds') );
```

Comment:
Good catch. It should be "at" midnight. I was going for 10ms after but probably should increase that a bit anyway since that's pretty close to on midnight and RH sources may not have reset yet.

Reply:
We can reset to unknown right at midnight UTC and then query sources after say, 15min. If anyone asks for RH location before we query that would go the "we don't know yet, query sources" branch anyway